<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectForm</class>
 <widget class="QDialog" name="NewProjectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="HeadingLbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create a Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ProjTitleLbl">
       <property name="text">
        <string>Project Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ProjTitle"/>
     </item>
     <item>
      <widget class="QLabel" name="DescLbl">
       <property name="text">
        <string>Project Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="ProjDesc"/>
     </item>
     <item>
      <widget class="QLabel" name="ManagerLbl">
       <property name="text">
        <string>Project Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ProjectManagerBox"/>
     </item>
     <item>
      <widget class="QLabel" name="MembersLbl">
       <property name="text">
        <string>Project Members</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AvailblUsrsLbl">
       <property name="text">
        <string>Available Users</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="AssignedUsersList"/>
       </item>
       <item>
        <widget class="QListWidget" name="AvailableUsersList"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="SrcUsers">
       <property name="placeholderText">
        <string>Search Available Users</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateProjectBtn">
       <property name="text">
        <string>Create Project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
